doc/news/_preparation_next_release.md

```diff
@@ -73,21 +118,23 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 
```

Comment:
Please make sure to write down the breaking changes in the Compatibility section. For the 1.0 release we will only go through this section to collect the changes.

Please also write about the changes in https://github.com/ElektraInitiative/libelektra/pull/3606.

Reply:
The keyname change actually wasn't listed under Compatibility either. I will add the `kdbOpen` and `keyCopy` stuff for the next release, but I think we should also summarize all the "Highlights" for 1.0 as well.

Reply:
> Maybe we should directly create a doc/news/_preparation_1.0_release.md?

A basic list of important changes in `doc/todo/RELEASE_1.0` might be better. If we start maintaining two separate versions of future release notes we could become out of sync. I think it may be better to just collect a list of all the important/breaking changes and then when it is time for the 1.0 release write one cohesive summary and search old release notes for details.

Reply:
Yes, doc/todo/RELEASE_1.0 for a list of breaking changes might be less confusing for developers.